src/examples/java/io/nats/examples/jetstream/NatsJsPushSub.java

```diff
@@ -81,14 +81,11 @@ public static void main(String[] args) {
             // A push subscription means the server will "push" us messages.
             // Build our subscription options. We'll create a durable subscription.
             // Durable means the server will remember where we are if we use that name.
-            PushSubscribeOptions.Builder builder = PushSubscribeOptions.builder();
-            if (exArgs.durable != null) {
-                builder.durable(exArgs.durable);
-            }
-            if (exArgs.deliver != null) {
-                builder.deliverSubject(exArgs.deliver);
-            }
-            PushSubscribeOptions so = builder.build();
+            PushSubscribeOptions so = PushSubscribeOptions.builder()
```

Comment:
Can you add a comment that null streams and durables are handled?

Reply:
done